.gitignore

```diff
@@ -19,6 +19,7 @@ htmlcov/
 # Jupyter
 .ipynb_checkpoints/
 *.ipynb  # Optional: if notebooks shouldn't be tracked
+examples/test_code.ipynb
```

Comment:
Ash to update this to ignore all .ipynb and make exception for this example notebook (if wanted)